physicalplan/unary/unary.go

```diff
@@ -24,6 +24,10 @@ type unaryNegation struct {
 	workers    worker.Group
 }
 
+func (u *unaryNegation) Explain() (me string, next []model.VectorOperator) {
+	return "[*unary]", []model.VectorOperator{u.next}
```

Comment:
Maybe this should be `unaryNegation` as per this [comment](https://github.com/thanos-community/promql-engine/pull/62#discussion_r989283683)?
```suggestion
	return "[*unaryNegation]", []model.VectorOperator{u.next}
```

Reply:
Makes sense, updated 👍